components/browser/search/src/main/java/mozilla/components/browser/search/SearchEngineManager.kt

```diff
@@ -94,8 +99,8 @@ class SearchEngineManager(
     private val localeChangedReceiver by lazy {
         object : BroadcastReceiver() {
             override fun onReceive(context: Context, intent: Intent?) {
-                launch(CommonPool) {
-                    load(context.applicationContext)
+                CoroutineScope(Dispatchers.Default).launch {
```

Comment:
Is there any difference (here and in the other places) between creating a scope with the default dispatcher and just using the `GlobalScope`? If not then I think we can replace those generic scope creations with just the global one until we actually make use of some local scope?